prototype2/tools/ReaderPcap.cpp

```diff
@@ -114,14 +157,17 @@ void ReaderPcap::printpacket(unsigned char *data, size_t len) {
   printf("\n");
 }
 
-void ReaderPcap::printstats() {
-  printf("rx packets %" PRIu64 "\n", stats.rx_pkt);
-  printf("  unknown  %" PRIu64 "\n", stats.eth_unkn);
-  printf("  arp      %" PRIu64 "\n", stats.eth_arp);
-  printf("  ipv4     %" PRIu64 "\n", stats.eth_ipv4);
-  printf("    udp    %" PRIu64 "\n", stats.ip_udp);
-  printf("    other  %" PRIu64 "\n", stats.ip_unkn);
-  printf("rx short   %" PRIu64 "\n", stats.rx_skipped);
-  printf("rx bytes   %" PRIu64 "\n", stats.rx_bytes);
+
+void ReaderPcap::printStats() {
+  printf("Total packets        %" PRIu64 "\n", Stats.PacketsTotal);
```

Comment:
Seeing as you have included the _fmt_ library, why not use the safer, simpler and faster way of printing these strings?
```suggestion
  fmt::print("Total packets        {}\n", Stats.PacketsTotal);
```